docs/api-docs/partner/Dev-portal-overview.md

```diff
@@ -0,0 +1,100 @@
+# Developer Portal Overview
+
+<div class="otp" id="no-index">
+
+### On this page
+- [About BigCommerce apps](#about-bigcommerce-apps)
+- [Use cases](#use-cases)
+- [Developing your app](#develop-your-app)
+- [Resources](#resources)
+</div> 
+
```

Comment:
Developer Portal is the resource where you can create, manage and publish your application for the BigCommerce App Marketplace.

Reply:
I added this line